components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -1117,33 +1193,10 @@ sql::InitStatus PublisherInfoDatabase::EnsureCurrentVersion() {
 
   const int old_version = meta_table_.GetVersionNumber();
   const int cur_version = GetCurrentVersion();
+  const int start_version = old_version + 1;
 
-  // to version 2
-  if (old_version < 2 && cur_version < 3) {
-    if (!MigrateV1toV2()) {
-      LOG(ERROR) << "DB: Error with MigrateV1toV2";
-    }
-  }
-
-  // to version 3
-  if (old_version < 3 && cur_version < 4) {
-    if (!MigrateV2toV3()) {
-      LOG(ERROR) << "DB: Error with MigrateV2toV3";
-    }
-  }
-
-  // to version 4
-  if (old_version < 4 && cur_version < 5) {
-    if (!MigrateV3toV4()) {
-      LOG(ERROR) << "DB: Error with MigrateV3toV4";
-    }
-  }
-
-  // to version 5
-  if (old_version < 5 && cur_version < 6) {
-    if (!MigrateV4toV5()) {
-      LOG(ERROR) << "DB: Error with MigrateV4toV5";
-    }
+  for (auto i = start_version; i <= cur_version; i++) {
+    Migrate(i);
   }
 
   meta_table_.SetVersionNumber(cur_version);
```

Comment:
nit: cur_version and CurrentVersion, would current_version be better for consistency

Reply:
done